src/components/Navigation/Aside/index.jsx

```diff
@@ -52,6 +53,8 @@ const Aside = ({ handleMobileToggle }) => {
     shallowEqual
   );
 
+  const usersMessage = useFormatMessage('Aside.users');
```

Comment:
Why are we storing the translation in a constant if we only use it from one place?

Reply:
React does not allow hook son conditions or nested functions thats why we store the translation in a constant., so even though we only use it once, we need to store it somewhere.